package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "5.5.2",
+  "version": "6.1.0",
```

Comment:
Please bump up the version to 6.1.1 since there already has been a release.

Reply:
Updated release and version to 6.1.1.